assgnment2/userdetail.json

```diff
@@ -0,0 +1 @@
+"[{\"userName\":\"maneesh\",\"userRollNumber\":1,\"userCources\":[\"A\",\"B\",\"C\",\"D\"],\"userAge\":24,\"userAddress\":\"kanpur\"},{\"userName\":\"manee\",\"userRollNumber\":2,\"userCources\":[\"A\",\"B\",\"C\",\"D\"],\"userAge\":6,\"userAddress\":\"kk\"}]"
```

Comment:
You shouldn't push generated files or files with any confidential data on GitHub. You can remove this file from version tracking by using `.gitignore`